include/feeding/action/MoveTowardsPerson.hpp

```diff
@@ -3,21 +3,19 @@
 
 #include <libada/Ada.hpp>
 
+#include "feeding/FeedingDemo.hpp"
 #include "feeding/Workspace.hpp"
 #include "feeding/perception/Perception.hpp"
 
 namespace feeding {
 namespace action {
 
 bool moveTowardsPerson(
-    const std::shared_ptr<ada::Ada>& ada,
     const aikido::constraint::dart::CollisionFreePtr& collisionFree,
     const std::shared_ptr<Perception>& perception,
     const ros::NodeHandle* nodeHandle,
```

Comment:
Does the FeedingDemo not have the NodeHandle?

Reply:
Good catch. Removed nodeHandle from all action interfaces